dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -372,6 +374,19 @@ const decideShowDefaultGate = (): ShowGateValues => {
 	return undefined;
 };
 
+const getGateDisplayCount = (): number => {
+	// TODO: retrieve the number of time the page has been displayed
+	const count = storage.local.get('gate_display_count') as number;
```

Comment:
shall we parse this properly like we do [here](https://github.com/guardian/dotcom-rendering/blob/1e0f774ab2db79de5dd39d67ec0e8cfac29aaf65/dotcom-rendering/src/lib/alreadyVisited.ts#L9)?

Reply:
Oh yes, good idea! One sec